content/kubermatic/main/installation/install-kkp-EE/_index.en.md

```diff
@@ -59,6 +59,13 @@ spec:
 
 Follow the CE install guide as normal, the remaining steps apply equally to the Enterprise Edition.
 
+### Included Application Catalog
+
+The EE Edition of KKP includes the ability to deploy an Application Catalog consisting of [well-known Kubernetes applications](https://github.com/kubermatic/kubermatic/tree/main/pkg/ee/default-application-catalog/applicationdefinitions).
+The catalogue provides an easy solution to make use of upstream helm charts after the installation. It get your organization up and running quickly. Applications are integrated into the KKP cluster lifecycle and can be directly managed via the UI, GitOps or KKP Cluster Templates. Afterwards, the initial catalog can be extended and tweaked to your liking. For more details please refer to the [Applications documentation]({{< ref "../../architecture/concept/kkp-concepts/applications/" >}}).
```

Comment:
Suggestion for minor changes: 
`The EE Edition of KKP includes the ability` should be `The Enterprise Edition(EE) of KKP offers the capability`
`The catalogue` should be `The catalog`
`It get your organization` should be `It gets your organization`
`tweaked to your liking` can be ` tweaked as per your preference`
`For more details please` should be `For more details, please`

Reply:
done